cmd/kube-proxy/app/server_others.go

```diff
@@ -555,28 +547,15 @@ func cidrTuple(cidrList string) [2]string {
 	return cidrs
 }
 
-func getProxyMode(proxyMode string, canUseIPVS bool) string {
+func getProxyMode(proxyMode string) string {
```

Comment:
maybe we can get rid of this since this should be handled by validation https://github.com/kubernetes/kubernetes/pull/111806/commits/a3923f8cc95f3f9d05c2baf7e6d0a69e9129391c#r946585562

Reply:
ah, I had gotten rid of it originally but the unit tests fail. Looks like that's a unit test problem though

Reply:
validation allows `""` which means "kube-proxy should pick the best available proxy" (so it doesn't actually get defaulted). So we need something.